modules/simple_bucket/README.md

```diff
@@ -0,0 +1,97 @@
+# Terraform Google Cloud Storage Module
+
+This module makes it easy to create a GCS bucket, and assign basic permissions on it to arbitrary users.
+
+The resources/services/activations/deletions that this module will create/trigger are:
+
+- One GCS buckets
+- Zero or more IAM bindings for those buckets
+
+## Compatibility
+
+ This module is meant for use with Terraform 0.12. If you haven't [upgraded](https://www.terraform.io/upgrade-guides/0-12.html)
+  and need a Terraform 0.11.x-compatible version of this module, the last released version intended for
```

Comment:
There's no version of this module for TF 0.11. Could probably drop this whole section.